benchmarks/benchmarks/bench_random.py

```diff
@@ -34,11 +34,11 @@ class Randint(Benchmark):
 
     def time_randint_fast(self):
         """Compare to uint32 below"""
-        np.random.randint(0, 2**30, size=10**5)
+        np.random.randint(low=0, high=2**30, size=10**5)
```

Comment:
I specifically wrote a test <a href="https://github.com/numpy/numpy/pull/7151/files#diff-061a4d644573e126c1d87ab55ad1f5e5R236">here</a> to address that concern.  Maybe it should be expanded, but I don't follow why you would want to have all of your other tests testing the old functionality.